regsync/regsync/registry/mlflow.py

```diff
@@ -2,17 +2,30 @@
 from typing import List, Dict
 from regsync.types import Model, ModelVersion, Artifact, LATEST_STAGE_NAME
 from regsync.registry import ModelRegistry
+from regsync.util import compress
 from mlflow.tracking import MlflowClient  # type: ignore
 from mlflow.entities.model_registry import RegisteredModel  # type: ignore
+import os
 
 logger = logging.getLogger(__name__)
 
 
 class Client(ModelRegistry):
-    def __init__(self, uri: str):
-        logger.info(f"Registry client initialized with uri '{uri}'")
-        self.client = MlflowClient(registry_uri=uri)
+    def __init__(self, uri: str, model_cache_path: str = "models"):
```

Comment:
I don't think this should have a default. This default could drift from the default in the controller code producing weird behavior.